app/view_models/auctions_index_view_model.rb

```diff
@@ -2,16 +2,19 @@ class AuctionsIndexViewModel
   attr_reader :auctions, :current_user
 
   def initialize(auctions:, current_user:)
-    @auctions = auctions.to_a
+    @auctions = auctions
     @current_user = current_user
   end
 
   def active_auction_count
-    auctions.count { |i| i.started_at < Time.current && Time.current < i.ended_at }
+    auctions
+      .where('started_at < ?', Time.current)
```

Comment:
We'll probably move this back to an `select` or `inject` when it no longer becomes an easy thing to count. And/or we should move this method to `Presenter::Auction` but I don't want to bikeshed this any more right now.